.github/workflows/create-release-branch.yml

```diff
@@ -67,6 +77,20 @@ jobs:
       - name: Ensure all package versions are consistent
         run: rush ensure-consistent-versions
 
+      # Update the types field in the package.json
```

Comment:
Doing this on the create-release-branch is a little odd as the release branch may be merged back into main. I'd like to keep this in however and we can update when we solidify the branching strategy